CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+**January 23, 2024 [DESCW-1945](https://apps.itsm.gov.bc.ca/jira/browse/DESCW-1945)**
```

Comment:
i thought we were going to start dropping the changelog

Reply:
We were starting now. lol